src/matter/common/ExchangeManager.ts

```diff
@@ -53,7 +53,7 @@ export class ExchangeManager<ContextT> {
     addNetInterface(netInterface: NetInterface) {
         this.netListeners.push(netInterface.onData((socket, data) => {
             this.onMessage(socket, data)
-                .catch(error => logger.error(error));
+                .catch(error => logger.error(error.stack));
```

Comment:
Maybe this should be modified in Logger.ts to always display the stack on .error() when the argument is of Error class?
I don't see a reason why not to always display the stack...

Reply:
Ok